src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -24,10 +35,11 @@ function cell_diffusion_timescale(closure::IsotropicDiffusivity, diffusivities,
 end
 
 function cell_diffusion_timescale(closure::AnisotropicDiffusivity, diffusivities, grid)
+    topo = topology(grid)
 
-    Δx = min_Δx(grid)
-    Δy = min_Δy(grid)
-    Δz = min_Δz(grid)
+    Δx = minimum_grid_spacing(grid.Δx, topo[1])
```

Comment:
If we use `minimum_grid_spacing`, can we delete `min_Δx` if its unused? We shouldn't leave a vestigal `min_Δx` function lying around.